packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue

```diff
@@ -277,4 +311,58 @@ export default {
     }
 }
 
+.c-search-btn-clear {
+    $btn-size: 52px;
+    $icon-size: 14px;
+
+    background: $white;
+    border: none;
+    width: $btn-size;
+    height: $btn-size;
+    right: 5px;
+    top: 5px;
+    position: absolute;
+    cursor: pointer;
+
+    svg {
+        width: $icon-size;
+        height: $icon-size;
+        top: 18px;
+        left: 19px;
+        position: absolute;
+
+        g {
+            fill: #535353;
```

Comment:
Can we use colour variable from fozzie-colour-palette here?

Reply:
Fixed.